wondrous-app/pages/collaboration/[username]/boards.lazy.tsx

```diff
@@ -385,6 +386,14 @@ function BoardsPage() {
     filters,
   });
 
+  useEffect(() => {
```

Comment:
is this type of pattern common?

Reply:
It’s a parent state update from the child, since our header requires data from the inside of the page we can either update it from here or make a new fetch request inside the header, but even then we still rely on the current entityType to render breadcrumbs. Another way to do this is to lift state and main request to parent component, this can work too but over time can be hard to manage imo, we can also move Header inside the page but we would get a unmount->mount on every page change and it would cause flickering on static elements inside the header

tldr:  I think it’s okay and the pattern is pretty common tho not ideal imo